cmd/agent-js-worker/src/agent.js

```diff
@@ -0,0 +1,1089 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
```

Comment:
is this file same as aries.js from aries-framework-go, then add a reference to original file.

Reply:
it will be extended with functionality from `trustbloc-agent-js-worker`

Reply:
then it shouldn't be called 'aries-js-worker' once merged

Reply:
it's **agent**-js-worker not **aries**-js-worker